.github/workflows/deploy_ensnode_subgraph.yml

```
@@ -0,0 +1,107 @@
+name: Build and deploy ENSNode
+
+on:
+  push:
+    branches:
+      - feat/railway-ensnode-deployments
+      - main
+      - alpha
+      - subgraph
+
+jobs:
+  image-build-and-push:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.determine_tag.outputs.tag }}
+      svc_id: ${{ steps.determine_tag.outputs.svc_id }}
+      docker_image_id: ${{ steps.determine_tag.outputs.docker_image_id }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Extract Branch Name
+        run: echo "BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+    
+      # We need to recognize which branch had push incomin
```

```suggestion
      # We need to recognize which branch had an incoming push
```

Is that correct? Just trying to fix the typo